server/athenian/api/align/goals/dbaccess.py

```diff
@@ -54,6 +64,49 @@ async def delete_goal(account_id: int, goal_id: int, sdb_conn: Connection) -> No
     await sdb_conn.execute(sa.delete(Goal).where(where_clause))
 
 
+async def delete_team_goals(
+    account_id: int, goal_id: int, team_ids: Sequence[int], sdb_conn: Connection,
+) -> None:
+    """Delete a set of TeamGoal-s from DB."""
+    assert team_ids
+    assert conn_in_transaction(sdb_conn)
+    await _validate_team_goal_deletions(account_id, goal_id, team_ids, sdb_conn)
+
+    where_clause = sa.and_(TeamGoal.goal_id == goal_id, TeamGoal.team_id.in_(team_ids))
+    delete_stmt = sa.delete(TeamGoal).where(where_clause)
```

Comment:
Formatting:
```suggestion
    where_clause =
    delete_stmt = sa.delete(TeamGoal).where(sa.and_(
        TeamGoal.goal_id == goal_id,
        TeamGoal.team_id.in_(team_ids),
    ))
```
we will never-ever inspect the where condition separately, so the variable is not needed. Also, this is how the existing code is formatted.